IM_calculation/scripts/split_IESD.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+import argparse
+from pathlib import Path
+
+g=9.81
```

Comment:
inconsistent g value within repo (different units) - I'd prefer to import it.

Reply:
File deleted